### ISSUE TITLE
Implement accordion functionality in modal for optional inputs

### ISSUE THREAD

**Issue by meganrm** [simularium/simularium-website#396]

**Comment by interim17** [comment]
design link shows accordion menu in a modal, but my understanding is we are actually building this full screen [like this](https://xd.adobe.com/view/5f448466-eebd-4ad2-bdd8-314a55133403-0a17/screen/dc823dee-f9ca-4b87-925c-e461f6180a8b/specs/)

**Comment by meganrm** [comment]
> design link shows accordion menu in a modal, but my understanding is we are actually building this full screen [like this](https://xd.adobe.com/view/5f448466-eebd-4ad2-bdd8-314a55133403-0a17/screen/dc823dee-f9ca-4b87-925c-e461f6180a8b/specs/)

Yes you're right, I hadn't updated this yet after the new designs
